tests/kernel/boot_ltp.pm

```diff
@@ -41,6 +40,25 @@ sub run {
     }
 
     select_serial_terminal;
+}
+
+sub run {
+    my ($self) = @_;
+    my $cmd_file = get_var('LTP_COMMAND_FILE') || '';
```

Comment:
What's usage for $cmd_file? I can not see any other place use this var in current file. Correct me if i am wrong.

Reply:
If you look carefully, this is not related to this PR and it is used in `schedule_tests`.